docs/api-reference/next.config.js/headers.md

```diff
@@ -26,6 +26,10 @@ Headers allow you to set custom HTTP headers for an incoming request path.
 To set custom HTTP headers you can use the `headers` key in `next.config.js`:
 
 ```js
+/**
+ * @type {import('next').NextConfig}
+ */
```

Comment:
Does it work directly on the `module.exports`? I think it needs to be set to a variable to allow the type to work correctly.